lib/resources/r4/financial_transaction_create.yaml

```diff
@@ -0,0 +1,325 @@
+      ]
+    }
+  url: https://fhir-ehr.cerner.com/r4/StructureDefinition/financial-transaction-transaction-alias
+
+- name: financial-transaction-transaction-alias-alias
+  required: 'Yes'
+  cardinality: 1..1
+  type: string
+  description: The aliases used to create the FinancialTransaction resources.
+  action: create
+  example: |
+    {
+      "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/financial-transaction-transaction-alias",
+      "extension": [
+          {
+              "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/financial-transaction-transaction-alias-alias",
+              "valueString": "729813"
+          }
+      ]
+    }
+  note: A value that correlates to a client defined name which represents the combination of the type, subtype, and reason describing the FinancialTransaction. Is required to create al FinancialTransaction resource
+  url: https://fhir-ehr.cerner.com/r4/StructureDefinition/financial-transaction-transaction-alias-alias
+
+- name: financial-transaction-location
+  required: 'No'
+  cardinality: 0..1
+  type: string
+  description: The location where the payment was collected.
+  action: create
+  example: |
+    {
+      "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/financial-transaction-location",
+      "valueString": "201802160003"
+    }
+  note: These are client configured location values.
+  url: https://fhir-ehr.cerner.com/r4/StructureDefinition/financial-transaction-location
+
+- name: financial-transaction-method
+  required: 'C'
+  cardinality: 0.1
+  type: CodeableConcept
+  description: The method of payment.
+  action: create
+  example: |
+    {
+      "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/financial-transaction-method",
+      "valueCodeableConcept": {
+          "coding": [
+              {
+                  "code": "CHCK",
+                    "display": "Check",
+                    "system": "http://terminology.hl7.org/CodeSystem/v2-0570",
+                    "userSelected": true
+              }
+          ],
+          "text": "Check"
+      }
+    }
+  note: Required on payment transactions. Payment methods of Cash, Credit Card, Check, Cashier’s Check, Traveler’s Check and Money Order will be supported for FinancialTransaction creates.
+  url: https://fhir-ehr.cerner.com/r4/StructureDefinition/financial-transaction-method
+
+- name: financial-transaction-card-brand
+  required: 'C'
+  cardinality: 0..1
+  type: string
+  description: The brand of credit card used for the transaction.
+  action: create
+  example: |
+    {
+      "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/financial-transaction-card-brand",
+      "valueString": "VISA"
+    }
+  note: Required on payment transactions when method is Credit Card. Acceptable values are “VISA”, “MASTER”, “AMEX”, “DISCOVER”.
+  url: https://fhir-ehr.cerner.com/r4/StructureDefinition/financial-transaction-card-brand
+
+- name: financial-transaction-account-number
```

Comment:
We arent supporting this right now

Reply:
https://github.com/cerner/fhir.cerner.com/pull/297/commits/a1a39411fbb35b2ccf79e521ffae5e6ae92d62b9#diff-26870e1b94fc9955e130f3f940dbe2e7L152-L165

Reply:
+1